content/concepts/networks-and-servers/amendments.md

```diff
@@ -62,6 +62,12 @@ The voting configuration of a `rippled` server has no impact on it becoming amen
 
 You can unblock amendment blocked servers by upgrading to the newest version of `rippled`.
 
+### Amendment Blocked in Clio Servers
+<a id="amendment-blocked-clio"></a>
```

Comment:
This isn't necessary; the ID in the previous paragraph exists because there are legacy release notes & blogs that link the old ID. As far as I know, there are no legacy links to this section so it doesn't need an additional ID.